src/index.js

```diff
@@ -23,6 +23,9 @@ addAction( 'native.render', 'gutenberg-mobile', ( props ) => {
 	setupJetpackEditor(
 		props.jetpackState || { blogId: 1, isJetpackActive: true }
 	);
+	if ( __DEV__ ) {
+		require( './block-experiments-setup' );
```

Comment:
I'm wondering why this needs to be required dynamically. Could we instead `import { registerBlock as registerLayoutGridBlock } from '../block-experiments/blocks/layout-grid/src';` here, and invoke it under `__DEV__`? If this is for the incidental idempotency given by `require.cache`, maybe we can explicitly make this idempotent by setting a flag in the file-level closure here?

Reply:
I believe that a `require` is necessary because metro doesn't support require imports just yet. See https://github.com/wordpress-mobile/gutenberg-mobile/pull/2582#discussion_r613574059

Reply:
That comment refers to why we would use `require()` instead of `import()`, which are both dynamic. What I'm wondering is, why are we aiming for this to be dynamic? I.e. why not a static import at the top of the file, and dynamically register the block under `__DEV__`?

Reply:
Good question! I think part of the reason why I included it like this is so that we don't load the js script when the flag is not set. but since we are not downloading the js I guess this part doesn't really matter.

Reply:
It would be cool to explore something like this down the road for lazy-loading / code splitting.